orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/utils/HealthHelper.groovy

```diff
@@ -82,6 +82,21 @@ class HealthHelper {
     } as Map
   }
 
+  static boolean someAreDown(Map instance, Collection<String> interestingHealthProviderNames) {
+    List<Map> healths = filterHealths(instance, interestingHealthProviderNames)
+
+    if (!interestingHealthProviderNames && !healths) {
```

Comment:
I know at one point in some places there was a difference between null interestingHealthProviders and an empty list of interesting health providers (null I think being 'use everything available' and empty being ('I don't care about health, just let it proceed'), will try to see if I can find where that was and if it is still a thing